Source/JavaScriptCore/runtime/ArrayBuffer.cpp

```diff
@@ -208,6 +208,14 @@ Ref<ArrayBuffer> ArrayBuffer::create(const void* source, size_t byteLength)
     return buffer.releaseNonNull();
 }
 
+Ref<ArrayBuffer> ArrayBuffer::create(std::span<uint8_t>&& span)
```

Comment:
We usually pass std::span by value, it is cheap to copy.

Reply:
I will change all of these, but there is pre-existing code in WebKit that uses r-value references for spans.

The thing I like about using a reference is that it can be explicitly cleared out so that if someone tries to use it afterwards, it will produce an error, rather than working "by accident".

Reply:
Because std::span<> doesn't own the memory, it is completely fine to keep using the span after calling this create() function though.